server/vulnerabilities/cpe_test.go

```diff
@@ -226,27 +225,10 @@ func TestSyncsCPEFromURL(t *testing.T) {
 	dbPath := path.Join(tempDir, "cpe.sqlite")
 
 	err := SyncCPEDatabase(
-		client, dbPath, config.FleetConfig{Vulnerabilities: config.VulnerabilitiesConfig{CPEDatabaseURL: ts.URL}})
+		client, dbPath, WithCPEURL(ts.URL))
 	require.NoError(t, err)
 
 	stored, err := ioutil.ReadFile(dbPath)
 	require.NoError(t, err)
 	assert.Equal(t, "Hello world!", string(stored))
 }
-
-func TestSyncsCPESkipsIfDisableSync(t *testing.T) {
```

Comment:
Why are we removing this?

Reply:
I refactored `SyncCPEDatabase` to not take the whole `config.FleetConfig`, and instead just define the args it needs.

One of them was `DisableDataSync`, I thought kind of pointless to have:
```go
func doX(..., notDoX bool) {
  // ...
}
```

This test was testing such (now removed) behavior.